financial-templates-lib/clients/TokenBalanceClient.js

```diff
@@ -28,23 +28,23 @@ class TokenBalanceClient {
   }
 
   // Delete all data within the client.
-  clearState = async () => {
```

Comment:
nit: this function format should be in traditional style (e.g. clearState() {}) since its at the class level